elements/pfe-band/src/pfe-band--lightdom.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../pfe-sass/pfe-sass";
+
+// For each section in the band layout
+@each $section in (aside, header, body, footer) {
+     [slot="pfe-band--#{$section}"] { // i.e., pfe-band--header
+      // Other browsers are getting grid-area through shadow-dom
+      // Only need to target edge in the lightdom
+      @supports (display: -ms-grid) {
+          // Apply the section as a named grid-area
+          grid-area: #{$section};
+      }
+    }
+}
+
+
+@media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
+    pfe-band > * > * {  // target children of body area
+      width: 51%; // over 50% so if multiple items in body they will stack
+      margin-right: 4.5%;
+      float: left;
+    }
+    pfe-band [slot="pfe-band--header"],[slot="pfe-band--footer"] { // reset width of children in header & footer
+      width: 55%; // This prevents asides aligned with header from display above the entire band
+      margin: 0;
+    }
+    pfe-band [slot="pfe-band--aside"] { // reset width of children in header & footer
+      width: 30%;
+      float: right;
+    }
+    &::after {
```

Comment:
I don't think this `&::after` is attached to anything at the moment, did you want it on `pfe-band::after`?

Reply:
Ah YES! whoops, I was targeting everything 🤦‍♀